braindecode/preprocessing/preprocess_classes.py

```diff
@@ -0,0 +1,117 @@
+"""Preprocessor objects based on mne methods.
```

Comment:
This file is going to be removed, am I right?

Reply:
yes @sliwy